framework/source/class/qx/data/SingleValueBinding.js

```diff
@@ -1185,14 +1185,14 @@ qx.Class.define("qx.data.SingleValueBinding",
         // go threw all added listeners (source)
         for (var i = 0; i < id.sources.length; i++) {
           // check if a source is available
-          if (id.sources[i]) {
+          if (id.sources[i] && id.listenerIds[i]) {
```

Comment:
@mever it would maybe help to add a this.error in the situation where somehow `sources` gets "out of sync" from `listenerIds`. we hesitate to merge this as the situation could be caused by a bug somewhere else which is then silenced by this pr. A this.error would help here as it offers a stack trace.